flask_mongorest/methods.py

```diff
@@ -1,17 +1,29 @@
+import typing
+
+
 class Create:
-    method = 'POST'
+    method = "POST"
+
 
 class Update:
-    method = 'PUT'
+    method = "PUT"
+
 
 class BulkUpdate:
-    method = 'PUT'
+    method = "PUT"
+
 
 class Fetch:
-    method = 'GET'
+    method = "GET"
+
 
 class List:
-    method = 'GET'
+    method = "GET"
+
 
 class Delete:
-    method = 'DELETE'
+    method = "DELETE"
+
+
+# type alias
+METHODS_TYPE = typing.Union[Create, Update, BulkUpdate, Fetch, List, Delete]
```

Comment:
Method Types should be enums in an ideal world. As written, the correct annotatation would be `Union[Type[A], Type[B], ...]` since we're referencing classes, not instances.